measure_extinction/extdata.py

```diff
@@ -20,15 +21,43 @@
 _poss_datasources = ["BAND", "IUE", "FUSE", "STIS", "SpeX_SXD", "SpeX_LXD", "IRS"]
 
 
-def _rebin(a, rebin_fac):
+def _rebin(waves, exts, rebin_fac):
     """
-    Hack code to rebin a 1d array
+    Code to rebin a 1d extinction curve
+
+    Parameters
+    ----------
+    waves : np.ndarray
+        the wavelengths of the extinction curve
+
+    exts : np.ndarray
+        the extinction values of the extinction curve
```

Comment:
Does this need to be extinction values?  Or could it also be for spectra?  Just wondering as would be useful to have a rebin for both.  No changes needed, just an idea that popped into my head when looking at the very nice changes.

Reply:
This could work for anything basically. I am just calling it extinction here because it is located in the extdata.py script. We could have a similar function in the stardata.py e.g.